python/ray/autoscaler/aws/defaults.yaml

```diff
@@ -43,38 +39,47 @@ auth:
 # configurations below.
 #    ssh_private_key: /path/to/your/key.pem
 
-# Provider-specific config for the head node, e.g. instance type. By default
-# Ray will auto-configure unspecified fields such as SubnetId and KeyName.
-# For more documentation on available fields, see:
-# http://boto3.readthedocs.io/en/latest/reference/services/ec2.html#EC2.ServiceResource.create_instances
+# Tell the autoscaler the allowed node types and the resources they provide.
+# The key is the name of the node type, which is just for debugging purposes.
+# The node config specifies the launch config and physical instance type.
+available_node_types:
+    ray.head.small:
+        node_config:
+          InstanceType: m5.large
+
+          # You can provision additional disk space with a conf as follows
+          BlockDeviceMappings:
+              - DeviceName: /dev/sda1
+                Ebs:
+                    VolumeSize: 100
+        resources: {"CPU": 2}
+        min_workers: 0
+        max_workers: 0
+    ray.worker.small:
+        node_config:
+          InstanceType: m5.large
+          ImageId: ami-0a2363a9cff180a64 # Deep Learning AMI (Ubuntu) Version 30
```

Comment:
```suggestion
          ImageId: latest_dlami # Deep Learning AMI (Ubuntu) Version 30
```

Reply:
Oh, I should probably remove this entirely because it is redundant (it is taken from worker_nodes)